src/node.ts

```diff
@@ -98,6 +105,7 @@ class Node extends VEventTarget {
     } else {
       output.setValue();
     }
+    this.dispatchEvent(new VEvent('setOutputValue'));
```

Comment:
The controller will have no info about which output was updated. The outputs emit their own events, so if the controller is interested in those events, it should subscribe to them directly. This might open a discussion about event bubbling that is not yet implemented with our VanillaEvent framework: the "nested" events could "bubble up" to node and network level, and the original event target would then be accessed via `event.target` property , while the `event.currentTarget` would point to the upper level object that has the listener attached.